cypress/integration/users-guide-link.js

```diff
@@ -0,0 +1,39 @@
+describe("A downloadable link for User's Guide", function() {
+
+  beforeEach(() => {
+    const email = Cypress.env("email");
+    const password = Cypress.env("password");
+    const baseUrl = Cypress.env("baseUrl");
+
+    cy.visit(baseUrl);
+
+    // it is ok for the email to be visible in the Command Log
+    expect(email, "email was set").to.be.a("string").and.not.be.empty;
+    // but the password value should not be shown
+    if (typeof password !== "string" || !password) {
+      throw new Error("Missing password value, set using CYPRESS_password=...");
+    }
+
+    cy.get('input[name="emaial"]')
```

Comment:
This should be using the custom command now

Reply:
Completed